nostalgia/ndf.py

```diff
@@ -378,7 +378,7 @@ def create_sample_data(self):
             .drop("index", axis=1)
             .drop("level_0", axis=1, errors="ignore")
         )
-        sample.to_parquet(fname)
+        sample.to_parquet(fname, allow_truncated_timestamps=True)
```

Comment:
Out of curiosity, what are truncated timestamps?

Reply:
https://stackoverflow.com/questions/53893554/transfer-and-write-parquet-with-python-and-pandas-got-timestamp-error